iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java

```diff
@@ -84,17 +87,26 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
         // computer
         
         AffOrgMatchComputer docOrgRelationMatchComputer = new AffOrgMatchComputer();
-        docOrgRelationMatchComputer.setAffOrgMatchVoters(of(
-                createNameCountryStrictMatchVoter(),
-                createNameStrictCountryLooseMatchVoter(),
-                createSectionedNameStrictCountryLooseMatchVoter(),
-                createSectionedNameLevenshteinCountryLooseMatchVoter(),
-                createSectionedShortNameStrictCountryLooseMatchVoter()));
+        docOrgRelationMatchComputer.setAffOrgMatchVoters(createDocOrgRelationMatcherVoters());
         
         // matcher
         
         return createAffOrgMatcher(docOrgRelationJoiner, docOrgRelationMatchComputer);
     }
+
+    /**
+     * Creates {@link AffOrgMatchVoter}s for {@link #createDocOrgRelationMatcher(JavaSparkContext, String, String, String, Float)} 
+     */
+    public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
+        return of(
```

Comment:
It seems more readable to use it as `ImmutableList.of(...)` like they do in the examples in the documentation.